ci-app/acm-repo/cluster/deployment-must-have-owner.yaml

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# [START constraint]
```

Comment:
was this being used? will changing this break a doc?

Reply:
1) yes, it's being used - https://cloud.google.com/anthos-config-management/docs/tutorials/app-policy-validation-ci-pipeline#validating_the_sample_app_configurations 
2) no it won't break the existing published doc because it still refers to the old sample repo, which still has the old `[constraint]` region tag. https://github.com/GoogleCloudPlatform/csp-config-management/blob/1.0.0/ci-app/acm-repo/cluster/deployment-must-have-owner.yaml#L14 

Will follow up with some docs updates to change the reference to this repo.